spec.bs

```diff
@@ -315,7 +315,7 @@ To <dfn>register an import map</dfn> given an {{HTMLScriptElement}} |element|:
       1. [=Report a warning to the console=] that an invalid address was given for the specifier key |specifierKey|; since |specifierKey| ended in a slash, so must the address.
       1. [=Continue=].
     1. Set |normalized|[|specifierKey|] to |addressURL|.
-  1. Return the result of [=map/sorting=] |normalized|, with an entry |a| being less than an entry |b| if |a|'s [=map/key=] is [=longer or code unit less than=] |b|'s [=map/key=].
+  1. Return the result of [=map/sorting=] |normalized|, with an entry |a| being less than an entry |b| if |b|'s [=map/key=] is [=code unit less than=] |a|'s [=map/key=].
```

Comment:
Are you making the assumption that a code unit outside the end of the string is taken to be zero and thus always less than any other code unit? That seems like something that shouldn't necessarily be an implicit assumption to me.

Reply:
https://infra.spec.whatwg.org/#code-unit-less-than explicitly handles prefixes (Steps 1 and 2) and we don't have to care about code unit outside the end of strings.

Reply:
Ah perfect, glad to see this is well-defined.